kuma/demos/helpers.py

```diff
@@ -394,9 +395,9 @@ def timesince(d, now=None):
                                  '%(number)d seconds ago', n))]
     if not now:
         if d.tzinfo:
-            now = datetime.datetime.now(LocalTimezone(d))
+            now = timezone.now(LocalTimezone(d))
```

Comment:
The timezone.now function doesn't work like this, you'll have to look into how to handle the case, probably by enabling that "local" timezone of the given datetime object for the duration of the creation of now. See the timezone.activate function.